tests/pyfunc/test_virtualenv.py

```diff
@@ -65,7 +65,7 @@ def temp_mlflow_env_root(tmp_path, monkeypatch):
 @use_temp_mlflow_env_root
 def test_restore_environment_with_virtualenv(sklearn_model):
     with mlflow.start_run():
-        model_info = mlflow.sklearn.log_model(sklearn_model.model, artifact_path="model")
+        model_info = mlflow.sklearn.log_model(sklearn_model.model, "model")
```

Comment:
All `log_model` calls specifying `artifact_path` as a positional argument will be transformed by this PR.

Reply:
I think this approach is a clever way to reduce unnecessary (and not fun) work.
Do you plan to replace them all in this PR or in a follow-up?

Reply:
Good question. In this PR, I'll only update the violating log_model calls in `tests`. I'll update the rest in a follow-up PR.

Reply:
SG!